packages/office-ui-fabric-react/src/components/Label/LabelCodepenButton.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
```

Comment:
This looks like it will be boilerplate all components will need. I don't think it is scalable to require so much boilerplate for every single component. Could you look into streamlining this?

Reply:
Agreed. THe actual 'button' should be in the example card and the component should just pass up the necessary information

Reply:
Moved everything into ExampleCard, deleting ...CodepenButton files entirely from the process. Does that work?